src/main/infrastructure/repository/HttpVendorListRepository.js

```diff
@@ -0,0 +1,36 @@
+import {VendorListRepository} from '../../domain/vendorlist/VendorListRepository'
+import {inject} from '../../core/ioc/ioc'
+import {HttpClient} from './http/HttpClient'
+import {VendorList} from '../../domain/vendorlist/VendorList'
+
+class HttpVendorListRepository extends VendorListRepository {
+  constructor({httpClient = inject(HttpClient)} = {}) {
+    super()
+    this._httpClient = httpClient
+  }
+
+  /**
+   * @param {Version} version
+   * @param {Language} language
+   * @returns {PromiseLike<VendorList>}
+   */
+  getVendorList({version, language}) {
+    const url = `${VENDOR_LIST_ENDPOINT}/${version.value}?language=${language.value}`
+    return this._httpClient
+      .get({url})
+      .then(response => response.data)
+      .then(
+        value =>
+          new VendorList({
+            version: value.vendorListVersion,
+            language: language || DEFAULT_VENDOR_LIST_VALUE_LANGUAGE,
+            value
+          })
+      )
+  }
+}
+
+const VENDOR_LIST_ENDPOINT = 'https://a.dcdn.es/borostcf/v2/vendorlist'
+const DEFAULT_VENDOR_LIST_VALUE_LANGUAGE = 'en'
```

Comment:
wouldn't be better to have this as a domain constant, now is repeated in two places, in the Language class and here.

Reply:
well seen, will refactor 👍